source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -487,9 +488,10 @@ namespace AdaptiveNamespace
         ComPtr<IAdaptiveElementRendererRegistration> elementRenderers;
         THROW_IF_FAILED(renderContext->get_ElementRenderers(&elementRenderers));
 
+        // For background image we use the default image renderer instead of custom image renderers
         ComPtr<IAdaptiveElementRenderer> elementRenderer;
-        THROW_IF_FAILED(elementRenderers->Get(HStringReference(L"Image").Get(), &elementRenderer));
-
+        THROW_IF_FAILED(MakeAndInitialize<AdaptiveNamespace::AdaptiveImageRenderer>(&elementRenderer));
```

Comment:
>THROW_IF_FAILED(MakeAndInitialize<AdaptiveNamespace::AdaptiveImageRenderer>(&elementRenderer)); [](start = 8, length = 95)

The problem with this is that the image renderer, unlike the other renderers, maintains a member which is the current xamlbuilder. The XamlBuilder, in turn, has members tracking things related to tracking the image loading for the Async case. (m_enableXamlImageHandling, m_listeners, m_imageLoadTracker, etc.). These won't be set properly on a new instance, and as a result i think the async call won't wait on completion of the images rendered here. I don't have a solution here, and that image async code could probably stand to be revisited, but we probably shouldn't make this change without making sure we understand the impact.